components/esm/cellref.cpp

```diff
@@ -4,13 +4,34 @@
 #include "esmreader.hpp"
 #include "esmwriter.hpp"
 
-void ESM::CellRef::load (ESMReader& esm, bool wideRefNum)
+void ESM::RefNum::load (ESMReader& esm, bool wide)
 {
-    loadId(esm, wideRefNum);
+    if (wide)
+        esm.getHNT (*this, "FRMR", 8);
+    else
+        esm.getHNT (mIndex, "FRMR");
+}
+
+void ESM::RefNum::save (ESMWriter &esm, bool wide, const std::string& tag) const
```

Comment:
This is confusing. Why does the save method need a tag argument, but the load method does not? It would make sense to use the same data structure for loading and saving.